symplyphysics/laws/thermodynamics/pressure_from_the_concentration_of_molecules_and_kinetic_energy.py

```diff
@@ -0,0 +1,34 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+
+## Ideal gas equation: p = 2/3 * n * E
+## Where:
+## p is pressure
+## n is concentration of molecules
+## E is the average kinetic energy of gas molecules
+
+## Conditions
+## The gas must be ideal
+
+average_kinetic_energy = Symbol("average_kinetic_energy", units.energy)
+molecules_concentration = Symbol("molecules_concentration", 1 / units.volume)
+pressure = Symbol("pressure", units.pressure)
+
+law = Eq(pressure, 2 / 3 * molecules_concentration * average_kinetic_energy)
```

Comment:
Derive it from https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/thermodynamics/average_kinetic_energy_of_molecules_from_temperature.py